tests/core/structure/sheet.test.ts

```diff
@@ -1,5 +1,6 @@
 import Sheet from "../../../src/core/structure/sheet";
 import Cell from "../../../src/core/structure/cell/cell.ts";
+import { CellState } from "../../../src/core/structure/cell/cellState.ts";
 
 describe("Sheet", () => {
   let sheet: Sheet;
```

Comment:
This test file needs to be broken down to different files. To me it seems like none of the issues are about `Sheet`